src/kv/commonKeyValueDB.ts

```diff
@@ -49,4 +49,6 @@ export interface CommonKeyValueDB {
   streamEntries: (table: string, limit?: number) => ReadableTyped<KeyValueDBTuple>
 
   count: (table: string) => Promise<number>
+
+  increment: (table: string, id: string, by?: number) => Promise<number>
```

Comment:
p3: Worth documenting (jsdoc) what the returning `number` means, I believe. We like to be extra clear, right?

Reply:
Also, since TypeScript interface doesn't have a tool to describe that default `by` will be 1 if omitted - let's add this info to the jsdoc.